app/components/candidate_interface/application_status_tag_component.rb

```diff
@@ -35,6 +35,15 @@ def colour
       end
     end
 
+    def provider_withdrew_application?
```

Comment:
As of yesterday, there's a method on `ApplicationChoice` for this https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/models/application_choice.rb#L177

Reply:
Great thanks for letting me know @steventux